pkg/executor/test/writetest/write_test.go

```diff
@@ -248,7 +248,7 @@ func TestIssue18681(t *testing.T) {
 	}()
 	sc.SetTypeFlags(oldTypeFlags.WithIgnoreTruncateErr(true))
 	tests := []testCase{
-		{[]byte("true\tfalse\t0\t1\n"), []string{"1|0|0|1"}, "Records: 1  Deleted: 0  Skipped: 0  Warnings: 0"},
+		{[]byte("true\tfalse\t0\t1\tb'1'\tb'1'\n"), []string{"1|1|1|1|1100010001001110011000100100111|1"}, "Records: 1  Deleted: 0  Skipped: 0  Warnings: 5"},
```

Comment:
As I've tested, the `LOAD DATA` in MySQL (both 5.6 and 8.0) also doesn't convert `"false"` to `0`. I don't know why it's originally implemented (in https://github.com/pingcap/tidb/pull/21310) as this.